sync-core/src/test/java/com/cloudant/sync/query/AbstractIndexTestBase.java

```diff
@@ -45,21 +45,23 @@ public void setUp() throws SQLException {
         assertThat(ds, is(notNullValue()));
         im = new IndexManager(ds);
         assertThat(im, is(notNullValue()));
-        db = im.getDatabase();
+        db = TestUtils.getDatabaseConnectionToExistingDb(im.getDatabase());
         assertThat(db, is(notNullValue()));
         assertThat(im.getQueue(), is(notNullValue()));
         String[] metadataTableList = new String[] { IndexManager.INDEX_METADATA_TABLE_NAME };
-        SQLDatabaseTestUtils.assertTablesExist(TestUtils.getDatabaseConnectionToExistingDb(db), metadataTableList);
+        SQLDatabaseTestUtils.assertTablesExist(TestUtils.getDatabaseConnectionToExistingDb(db),
```

Comment:
Done everywhere except in AbstractIndexTestBase https://github.com/cloudant/sync-android/commit/f2859642cda063d9ad46b8a6bff2067c6678c74b since doing so causes an issue with the IndexUpdaterTest.  Filed a separate issue to revisit the problem in the IndexUpdaterTest.  https://cloudant.fogbugz.com/f/cases/44055/Cloudant-Query-Investigate-IndexUpdaterTest-need-for-TestUtils-getDatabaseConnectionToExistingDb-db